packages/vscode-extension/src/webview/views/CreateDeviceView.tsx

```diff
@@ -63,31 +65,36 @@ function CreateDeviceView({ onCreate, onCancel }: CreateDeviceViewProps) {
   const [loading, setLoading] = useState<boolean>(false);
 
   const supportedDevices = useSupportedDevices();
-  const { iOSRuntimes, androidImages, deviceManager, reload } = useDevices();
 
-  useEffect(() => {
-    reload();
-  }, []);
+  const { project } = useProject();
+
+  const store$ = useStore();
+  const iOSRuntimes = use$(store$.devicesState.iOSRuntimes);
+  const androidImages = use$(store$.devicesState.androidImages);
```

Comment:
```suggestion
  const iOSRuntimes = use$(store$.devicesState.iOSRuntimes) ?? [];
  const androidImages = use$(store$.devicesState.androidImages) ?? [];
```
Why does it return `null` anyway? Does that happen during loading? Maybe we should use `[]` as the initial value?